File: ServiceBus/MessagingExpress/Topic/send-message.js
const { ServiceBusClient } = require("@azure/service-bus");

const connectionString = ""
const serviceBusClient = new ServiceBusClient(connectionString);

async function sendMessage(topicName){
    try
    {
        // create sender for topic
        const sender = serviceBusClient.createSender(topicName);
        // create message object
        const message = {
            body: "Hello, from Node to Topic.", 
            label: "greeting", 
            useProperties: { priority: 1}
        };

        //send message to topic
        await sender.sendMessages(message);
        await sender.close();
    }
    catch(err){
        console.log(err);
    }
    finally {
        await serviceBusClient.close();
    }
}

await sendMessage("topic001");

File: ServiceBus/MessagingPython/Topic/send-message.py
from azure.servicebus import ServiceBusClient, ServiceBusMessage

connection_str = ""
topic_name = "topic001"

def send_message(connection_str, topic_name):
    servicebus_client = ServiceBusClient.from_connection_string(connection_str)
    sender = servicebus_client.get_topic_sender(topic_name)
    message = ServiceBusMessage("Hello from Python")
    sender.send_messages(message)
    sender.close()
    servicebus_client.close()

send_message(connection_str, topic_name)
